api.py

```diff
@@ -59,10 +76,40 @@ def get(self, ipv4: str):
 
         return result
 
+
+    def _get_geoip2_service(self, ipv4: str):
+        client = geoip2.webservice.Client(
+            GEOIP2_USER_ID,
+            GEOIP2_LICENSE_KEY,
+        )
```

Comment:
```suggestion
        client = geoip2.webservice.Client(GEOIP2_USER_ID, GEOIP2_LICENSE_KEY)
```

Reply:
i run black and isort already and still make this formatting